src/partinverse.jl

```diff
@@ -0,0 +1,38 @@
+# default methods for when an inverse exists
+export r_inv, l_inv, retraction, coretraction
+
+"""
+    r_inv(function)
+    retraction(function)
+
+Return the left inverse of a function, i.e. a function that "undoes" `f`. Formally, 
+    ``\\text{retraction}(f)(f(x)) = x``
+"""
+r_inv(args...; kwargs...) = inverse(args...; kwargs...)
+
+"""
+    l_inv(function)
```

Comment:
Same hee, I think we should use only the name
```suggestion
    left_inverse(function)
```